app/controllers/lessons_controller.rb

```diff
@@ -4,4 +4,33 @@ class LessonsController < ApplicationController
   def show
     @lesson = Lesson.find(params[:id])
   end
+
+  def create
+    @course = Course.find(params[:course_id])
+    @lesson = Lesson.new(lesson_params)
+    @lesson.course = @course
+    if @lesson.save
+      flash[:notice] = " Woohoo! You've just added a lesson."
+      redirect_to course_lesson_path(@course, @lesson)
+    else
+      flash.now[:error] = "Ooops...Something is not right."
+      flash.now[:notice] = @course.errors.full_messages
+      render 'new'
+    end
+  end
+
+  def new
+    @course = Course.find(params[:course_id])
+    @lesson = Lesson.new
+  end
+
+  private
+
+  def lesson_params
+    params.require(:lesson).permit(
+    :title,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the start of the previous line.